app/models/application_reference.rb

```diff
@@ -3,14 +3,6 @@ class ApplicationReference < ApplicationRecord
 
   self.table_name = 'references'
 
-  validates :name, presence: true, length: { minimum: 2, maximum: 200 }, unless: -> { FeatureFlag.active?('decoupled_references') }
-  validates :email_address, presence: true,
-                            email_address: true,
-                            length: { maximum: 100 },
-                            uniqueness: { case_sensitive: false, scope: :application_form_id },
-                            unless: -> { FeatureFlag.active?('decoupled_references') }
-  validate :email_address_not_own
-  validates :relationship, presence: true, word_count: { maximum: 50 }, unless: -> { FeatureFlag.active?('decoupled_references') }
```

Comment:
I think there are some orphaned validation messages you can also remove from the yml locale files.

Reply:
Will do! Thanks for all the reviews today 👍